### PR TITLE
update release notes

### PR DESCRIPTION
**Pull request by devkral**
update release notes

### CODE REVIEW

**Comment by tarsil** [comment]
@devkral i Will approve. What do you need to do? Copy the release notes in a new release on GitHub after you create a tag 0.10.1 

Once you create the tag 0.10.1 push it to GitHub and automatically it will publish a new release 

Then create the release on GitHub like we so here where title is Version 0.10.1

And the notes are the copy of the release notes 🙂

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 0.10.1
+
+### Added
+- Decorated functions have now the `asyncz_tasks` attribute if Tasks are used in decorator mode without a provided `id`.
```
Comment (tarsil) [inline docs/release-notes.md:6]: Add a space between the title and the text

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+### Fixed
+- Fix backward compatibility in passing functions via fn keyword to add_task
```
Comment (tarsil) [inline docs/release-notes.md:9]: Same here

**Review by tarsil — CHANGES_REQUESTED** [review]

**Review by tarsil — APPROVED** [review]

**Comment by devkral** [comment]
you forgot a step: `__version__`

**Comment by tarsil** [comment]
Correct. It's in the docs actually but yes

**Pull request merged**